packages/nextjs/components/scaffold-eth/Address.tsx

```diff
@@ -92,50 +92,56 @@ export const Address = ({ address, disableAddressLink, format, size = "base" }:
 
   return (
     <div className="flex items-center">
-      <div className="flex-shrink-0">
-        <BlockieAvatar
-          address={checkSumAddress}
-          ensImage={ensAvatar}
-          size={(blockieSizeMap[size] * 24) / blockieSizeMap["base"]}
-        />
+      <div className="flex flex-col xs:flex-row items-center">
+        <div className="flex flex-row">
```

Comment:
I think we don't want the copy icon to go do a line wrap. We just want to not shrink. Probably you just need to add `flex-shrink-0` to the parent div.

In the particular case of the homepage (in your video, "connected address" + Address component) we'd just need to do flex-col on small resolutions and flex-row for the rest.